src/services/helpers.ts

```diff
@@ -241,3 +241,8 @@ export const splitTextBySearchString = (allText: string, highlight: string): [st
     allText.slice(indexAfterHighlight),
   ]
 }
+
+export const searchProfessions = (disciplines: Discipline[] | undefined, searchKey: string): Discipline[] | undefined =>
+  disciplines?.filter(discipline =>
+    normalizeStrings(discipline.title).toLowerCase().includes(normalizeStrings(searchKey.toLowerCase())),
```

Comment:
:upside_down_face: This does not do `trim()` the search string. We could introduce a custom helper as in Integreat to handle everything and reuse it:
``` ts
export const normalizeString = (str: string): string => normalizeStrings(str).toLowerCase().trim()
```